src/blocks/change-address/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { Edit } from './edit';
+import { save } from './save';
```

Comment:
Yes you are right, but that's how the[ @wordpress/create-block](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/) package generates those files, so i  just  left them as they were.

Reply:
Indeed this is the default way of @wordpress/create-block. However I'm used to set `save` as default export, so you don't need to import as `{ save }`, but this is not a blocker nor a requirement.

Reply:
Yes i agree  that when a file  is only exporting a variable better to use default export. Thats how i initally  was using it, but there  were build errors (default wasn't found) so i changed it in order to avoid debugging the  npm dependencies as part  of this feature